client/src/components/PhotoModalCarousel.jsx

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 import Photo from './Photo.jsx';
+import '../../dist/styles.css';
 
-const PhotoModalCarousel = (props) => (
-  <div className="photoModalCarousel">
-    {props.photos.map((photo, i) => {
-      const photoClassName = i === props.currentPhoto ?
-        'carouselPhoto' : 'carouselPhotoActive';
-      return <Photo photoClass={photoClassName} photoUrl={photo} key={i} />
-    })}
-  </div>
-);
+const PhotoModalCarousel = (props) => {
+  let carouselPhotos = props.photos.map((photo, i) => {
+    const photoClassName = i === props.currentPhoto ?
+      'carouselPhoto' : 'carouselPhotoActive';
+      return <Photo photoClass={photoClassName} photo={photo} key={i} />
+  })
+
+  let offsetAmount = ((100/props.photos.length) * 2 + (props.currentPhoto *
+    (100/props.photos.length)) * -1);
+
+  return (
+    <div className="photoModalCarousel" style={{
```

Comment:
Make a variable outside of the render function for the object in your style prop, then just feed the variable name into the style prop.